fermenter-mda/pom.xml

```diff
@@ -124,6 +136,12 @@
             <version>4.0.0-alpha-2</version>
             <scope>test</scope>
         </dependency>
```

Comment:
A (Administrative): add a comment to explain why we need 4.13.2 when we are otherwise using JUnit Jupiter